calico-cloud/operations/cluster-management.mdx

```diff
@@ -16,15 +16,16 @@ If you inadvertenty delete a managed cluster, rerun the `curl` command and the c
 
 ### Access your managed clusters
 
-Each time you install {{prodname}} on a cluster, the cluster is added to the drop-down menu in Manager.
+Each time you install {{prodname}} on a cluster, the cluster is added to the drop-down menu in Manager UI.
 
 ![cluster-menu](/img/calico-cloud/cluster-menu.png)
 
-Use this menu to move between your managed clusters. You can create as many managed clusters as your subscriptions allows.
+Use this menu to move between your clusters. You can connect clusters as your subscriptions allows.
```

Comment:
IMO. I think the statement should read as "You can connect as many managed clusters as your subscriptions allows."
As the limitation of the subscription would be on the number

Reply:
Agreed. Changed.